config/locales/views/campagnes.yml

```
@@ -5,7 +5,8 @@ fr:
         explication_campagne:  |
           **Une campagne eva est une sélection d'activités, de jeux, de questionnaires, formant un programme d'évaluation commun à plusieurs personnes.**
 
-          Au sein d'une campagne, plusieurs évaluations peuvent être regroupées par type de programme, par public cible, par dispositif d'accompagnement, etc.<br>(ex. : "Pré-positionnement 2022" ; "CEJ Sept. 2023" ; "Services civiques 1er trimestre").
+          Au sein d'une campagne, plusieurs évaluations peuvent être regroupées par type de programme, par public cible, par dispositif d'accompagnement, etc.
+          (ex. : "Pré-positionnement 2022" ; "CEJ Sept. 2023" ; "Services civiques 1er trimestre").
 
           Les bénéficiaires auront besoin du code campagne généré automatiquement pour démarrer leur évaluation.
 
```

```suggestion
          Les bénéficiaires auront besoin du code campagne généré automatiquement pour démarrer la passation.

```
Il y a une autre modification présente dans le texte de référence qui n'est pas souligné. À vérifier donc je pense pour avoir la source de vérité ?